examples/spring/build.gradle

```diff
@@ -19,5 +19,5 @@ repositories {
 dependencies {
     compile("org.springframework.boot:spring-boot-starter")
 
-    compile rootProject
+    compile project(":bugsnag-java")
```

Comment:
Assuming at a later date we would switch to using bugsnag-spring here

Reply:
Correct